### PR TITLE
Remove job-dsl from support plugin

### PR DESCRIPTION
**Pull request by casz**
### Your checklist for this pull request

🚨 Please review the [guidelines for contributing](../blob/master/docs/CONTRIBUTING.md) to this repository.

- [x] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side) and not your master branch!

- [x] Please describe what you did

- [x] Link to relevant GitHub issues or pull requests

- [ ] Link to relevant [Jenkins JIRA issues](https://issues.jenkins-ci.org)

- [x] Did you provide a test-case? That demonstrates feature works or fixes the issue.

### Description

@daspilker :raised_hands: `1.73-SNAPSHOT` got foobar since I accidentally only released the job-dsl plugin so had to bump the SNAPSHOT to `1.73.1-SNAPSHOT` to get it to work.

https://github.com/jenkinsci/job-dsl-plugin/pull/1173

### CODE REVIEW

File: integrations/pom.xml [hunk 1]
```diff
@@ -221,7 +221,7 @@
     <dependency>
       <groupId>org.jenkins-ci.plugins</groupId>
       <artifactId>job-dsl</artifactId>
-      <version>1.72</version>
+      <version>1.73.1-SNAPSHOT</version>
```
Comment (timja) [inline integrations/pom.xml]: normally a timestamped snapshot if no incrementals so you know what its running 😉

Comment (casz) [reply integrations/pom.xml]: I tried... job-dsl-core would not restore with timestamped

Comment (casz) [reply integrations/pom.xml]: I tried... job-dsl-core would not restore with timestamped (hint they don't have the same timestamp) @daspilker (@sghill current maintainer on gradle-jpi-plugin)

Comment (sghill) [reply integrations/pom.xml]: Is the issue that there are multiple modules and they end up with slightly different timestamps in the manifest?

I like creating release candidates for this scenario. Gradle-jpi-plugin will [take the given version unless it ends in `-SNAPSHOT`](https://github.com/jenkinsci/gradle-jpi-plugin/blob/master/src/main/java/org/jenkinsci/gradle/plugins/jpi/internal/VersionCalculator.java#L31), so it's possible to create a `1.73.0-rc.1` if you'd like. The `nebula.release` plugin makes this easy with its [`candidate` task](https://github.com/nebula-plugins/nebula-release-plugin#tasks-provided).

Comment (casz) [reply integrations/pom.xml]: > Is the issue that there are multiple modules and they end up with slightly different timestamps in the manifest?

I believe it is. You can explore the pom here: https://repo.jenkins-ci.org/webapp/#/search/quick/eyJzZWFyY2giOiJxdWljayIsInF1ZXJ5Ijoiam9iLWRzbCotMS43My4xKi5wb20ifQ==
You can hover over the artifact and download/view it.

**Comment by timja** [comment]
job-dsl is now released with support, good to 🚢  @casz?

**Comment by casz** [comment]
Once checks passes 😊

**Comment by timja** [comment]
travis passed =/ jenkins 1 error was java 11 java.net cert issue and another one was jvm dying randomly

**Comment by casz** [comment]
Well you have to solve the conflict 😆

**Comment by fwilhe** [comment]
Hello @casz, I've came here via the release notes, and just wanted to confirm if this changes anything in the user interface. I've looked into the linked PR in job-dsl, and to me it _looks_ like nothing changes from a user perspective, the same config still works. Is this the case?

Thanks

**Comment by casz** [comment]
As long as you install 1.74 of job-dsl nothing has changed for the user perspective

**Comment by fwilhe** [comment]
> As long as you install 1.74 of job-dsl nothing has changed for the user perspective

Good to know, thank you.

**Pull request merged**